inst/extdata/emissions/FUT_EMISS_GV.csv

```diff
@@ -0,0 +1,171 @@
+# File: FUT_EMISS_GV.csv
+# Title: Future atmospheric abundances and climate forcings from scenarios of global and regional hydrofluorocarbon
+# Units: "NA"
```

Comment:
@cahartin the emissions are in (GtCO2-eq yr -1) but we only use the emission factors, which I think are unit less but I'm not sure. I skimmed through the article we cite as a source and it didn't mention any units for emission factors.